backend/src/main/kotlin/xyz/elitese/ehrenamtskarte/database/repos/AcceptingStoresRepository.kt

```diff
@@ -8,7 +8,7 @@ import xyz.elitese.ehrenamtskarte.webservice.schema.types.Coordinates
 object AcceptingStoresRepository {
 
     fun findByIds(ids: List<Int>) =
-        AcceptingStoreEntity.find { AcceptingStores.id inList ids }.associateWithKeys({ it.id.value }, ids)
+        AcceptingStoreEntity.find { AcceptingStores.id inList ids }.sortByKeys({ it.id.value }, ids)
 
 
     // TODO would be great to support combinations like "Tür an Tür Augsburg"
```

Comment:
Jap, would be nice 👍 
On the other hand the code is not a issue board.

Reply:
I didn't add the todo